src/libraries/System.Net.Http/tests/StressTests/HttpStress/ClientOperations.cs

```diff
@@ -508,12 +508,6 @@ public static class ClientOperations
 
         private static void ValidateStatusCode(HttpResponseMessage m, HttpStatusCode expectedStatus = HttpStatusCode.OK)
         {
-            // [ActiveIssue("https://github.com/dotnet/runtime/issues/55261")]
-            if (m.StatusCode == HttpStatusCode.InternalServerError)
```

Comment:
We also need to remove the catch side:

https://github.com/dotnet/runtime/blob/cda0b74ce3db4b13675c5146f978a96410f2d9f6/src/libraries/System.Net.Http/tests/StressTests/HttpStress/StressClient.cs#L212-L216

Reply:
Fixed